Swift/QtUI/QtChatWindow.cpp

```diff
@@ -92,6 +92,11 @@ QtChatWindow::QtChatWindow(const QString& contact, QtChatTheme* theme, UIEventSt
 	subjectLayout_->addWidget(subject_);
 	setSubject("");
 	subject_->setReadOnly(true);
+	
+	bookmarkStar_ = new QPushButton(this);
+	connect(bookmarkStar_, SIGNAL(clicked()), this, SLOT(onBookmarkStarClicked()));
```

Comment:
Our convention for slot names is a 'handle' prefix instead of 'on'. See the other slots used in this file.